packages/itwin/breakdown-trees/src/Views/visibility/DataLink.ts

```diff
@@ -10,100 +10,86 @@ export class DataLink {
   public static async queryAllBuildings(iModel: IModelConnection) {
```

Comment:
> public static async queryAllBuildings

If you don't await anything, than you don't need to write async. Great job with the refactoring!

Reply:
Lint is giving error if I don't async

Reply:
You can see in my first commit I didn't added async but was getting lint errors so I added them back